openapi_test.go

```diff
@@ -201,6 +216,17 @@ func Test_tagFromType(t *testing.T) {
 			}
 		})
 	}
+
+	t.Run("struct with nested tags", func(t *testing.T) {
+		s := NewServer()
+		tag := SchemaTagFromType(s.OpenAPI, MyStructWithNested{})
+		nestedProperty := tag.Value.Properties["nested"]
+		require.Equal(t, "my struct", nestedProperty.Value.Description)
```

Comment:
Pleawtry to avoid panic on error

```suggestion
		tag := SchemaTagFromType(s.OpenAPI, MyStructWithNested{})
		require.Contains(t, tag.Value.Properties, "nested")
		nestedProperty := tag.Value.Properties["nested"]
		require.Equal(t, "my struct", nestedProperty.Value.Description)
```

Reply:
I don't understand the ask.

Reply:
Oh. Okay. Are  just saying check if tag is nil?

Reply:
I believe this concern is covered now with a fix from a previous concern

```golang
nestedProperty := tag.Value.Properties["nested"]
require.NotNil(t, nestedProperty)
```